package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1192,33 +1196,34 @@ def total_mass(group, compound='group'):
         ('total_mass', total_mass))
 
     @warn_if_not_unique
-    @check_pbc_and_unwrap
-    def moment_of_inertia(group, pbc=False, **kwargs):
+    @_pbc_to_wrap
+    @check_wrap_and_unwrap
+    def moment_of_inertia(group, wrap=False, **kwargs):
         """Tensor moment of inertia relative to center of mass as 3x3 numpy
         array.
 
         Parameters
         ----------
-        pbc : bool, optional
+        wrap : bool, optional
```

Comment:
Could you please document the `unwrap` and `compound` parameters that were added in 0.20.0? I know, it's not your fault. But at the moment the docs are confusing, especially as both are not even explicit (and hidden as kwarg).  Making them explicit in the signature would also be good.

Reply:
Can you add the missing docs here or should I raise a new issue?